modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_side_panel.scss

```diff
@@ -101,7 +101,7 @@
 		top: 0;
 		transition: top ease-in-out 0.1s, right ease-in-out 0.1s,
 			opacity ease-in 0.2s;
-		z-index: 1;
+		z-index: 10;
```

Comment:
Even with this value, X icon may not be visible as it will show on top of the panel content.

Reply:
For a quick bugfix, this is fine. A better solution would be to have no overlap, but I guess that would require redesign, or a significant refactoring or markup. We can consider this later on.